spacy/language.py

```diff
@@ -1363,7 +1362,7 @@ def resume_training(self, *, sgd: Optional[Optimizer] = None) -> Optimizer:
 
     def set_error_handler(
         self,
-        error_handler: Callable[[str, "Pipe", List[Doc], Exception], NoReturn],
+        error_handler: Callable[[str, PipeCallable, List[Doc], Exception], NoReturn],
```

Comment:
The docstring here is an example of one where the function signature in the docstring and the actual one no longer match. Do we want to update this?